commands/a_activity/password-stamp.js

```diff
@@ -107,10 +108,13 @@ module.exports = class DistributeStamp extends PermissionCommand {
                 pwdCollector.on('collect', async m => {
                     //update role and stop collecting if password matches
                     if (m.content === password) {
```

Comment:
@mwang2000 Lets toLowerCase() these two to make our hackers' life easier hahahah.